### ISSUE TITLE
After the latest node upgrade on azure devops , lerna with an older version 3.22.1 fails to work. Its considering the latest version 6.5.1 

### ISSUE THREAD

**Issue by geetikaharlalka22** [actions/runner-images#7158]
### Description

After the latest upgrade , the build pipelines are breaking with  an older node version 10.16.3 and lerna 3.22.1 . The global packages have the latest version and refuse to get uninstalled and run on the specific version . This is causing the build pipeline to break as our code is incompatible with the new versions . 

Request to provide a way to get this working to unblock us 

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [X] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

![image](https://user-images.githubusercontent.com/56873325/220170707-d57927da-7aa8-416a-a092-7d067981ec4e.png)

The Image where this does work 

![image](https://user-images.githubusercontent.com/56873325/220171200-43974d98-2b1e-4f7c-82fa-0aa706094b0b.png)

### Is it regression?

yes

### Expected behavior

If we try to use the node 10.16.3  and lerna 3.22.1 version it should install and allow the system to use the specified version rather than imposing the latest version 

### Actual behavior

It always uses lerna 6.5.1 even if you try to uninstall and install the version you require i.e 3.22.1 

### Repro steps

Have a task to install node 10.16.3 
Then add a task to install the lerna version 3.22.1 in monorepo directory
add another powershell task and try to execute the script 'lerna bootstrap --hoist" . It throws the below error 
![image](https://user-images.githubusercontent.com/56873325/220172013-27a74d1d-8e3f-4e1d-9ca5-91248b757f20.png)

**Comment by mikhailkoliada** [comment]
Duplicate of #7002, you may wish to use setup-node to specify a version you need.

**Comment by geetikaharlalka22** [comment]
@mikhailkoliada  We have tried to use the setup-node to specify the version we need 
![image](https://user-images.githubusercontent.com/56873325/220361304-154a52d2-8798-4bad-a33d-dabc760ae17f.png)
but even after that we are getting the error with Lerna as it has packages globally for 6.5.1 version which is creating the problem . 

Request you to please not close the issue
